### DISCUSSION TITLE
Question about decoy generation

### DISCUSSION THREAD

**Discussion by cjhjhj** [vdemichev/DiaNN#255] [Q&A]
Hello Vadim,

In a previous discussion, I could find the decoy generation rule for DIA-NN as follows,

Decoys are generated via GAVLIFMPWSCTYHKRQEND to LLLVVLLLLTSSSSLLNDQE mutation pattern for residues adjacent to the peptide termini. If such a residue is modified DIA-NN tries to mutate an adjacent residue farthest from the terminus, and if that one is modified too, then the terminal residue. And if the terminal is modified, then the initial residue is mutated, regardless of the presence of a modification. Examples:
PEPTIDE -> PDPTIEE
PC(cam)PTIDE -> PC(cam)LTIEE
PC(cam)M(ox)TIDE -> LC(cam)M(ox)TIEE
(acetyl)PC(cam)M(ox)TIDE -> (acetyl)PS(cam)M(ox)TIEE

Could you please let me know what is the rationale of the mutation rule? According to the rule, some AAs correspond to the same mutation (e.g., both G and A are mutated to L in a decoy). Is it okay to use such a mutation mapping (not one-to-one)?

Thank you.

**Comment by vdemichev** [comment]
Hi, this decoy generation just works well empirically. There's no need for one-to-one mutation mapping.

Vadim
